README.md

```diff
@@ -138,8 +137,6 @@ Local demos consist of HTML, CSS and JS (if Sass & JS exists), and are created i
 
 _(Sourcemaps aren't generated as this feature is incompatible with csso. We will revisit this when [gulp-ruby-sass](https://github.com/sindresorhus/gulp-ruby-sass) 1.0 is released)_
```

Comment:
Feel free to edit `csso` into `clean-css` (I'd do it but that might generate a merge conflict)